changelog.d/pr-10469

```diff
@@ -0,0 +1,10 @@
+synopsis: Deprecate `cabal format`
+packages: cabal-install
+prs: #10469
+significance:

```

Comment:
This is a significant change, please add the `significant` metadata to this field.